http-clients/url-connection-client/src/main/java/software/amazon/awssdk/http/urlconnection/UrlConnectionHttpClient.java

```diff
@@ -70,12 +75,16 @@
 
     private static final Logger log = Logger.loggerFor(UrlConnectionHttpClient.class);
     private static final String CLIENT_NAME = "UrlConnection";
+    private static final AttributeMap.Key<ProxyConfiguration> PROXY_CONFIGURATION =
```

Comment:
To be consistent with other http client lets assign the ProxyConfiguration directly to the class Field as done in other client constructors , I didnot get the benefit of assigning it to a AttributeMap and then doing a get.

Can we do something similar to [ApacheHttpClient](https://github.com/aws/aws-sdk-java-v2/blob/b673b8456a5e6c38132d5764661f64b1dcb981a6/http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java#L453) ?